### DISCUSSION TITLE
The external monitor does not work properly when it is connected to the laptop's usb-c thunderbolt port (with displayport).

### DISCUSSION THREAD

**Discussion by iamcalegari** [elementary/triage#239] [Q&A]
The external monitor does not work properly when it is connected to the laptop's usb-c thunderbolt port (with displayport).
The screen has a sort of green filter, but the mouse cursor doesn't.

I really want to use Elementary OS as the main system, but I can't...

The fun fact is that in the installation setup everything works great.

https://github.com/elementary/triage/assets/64042095/486d303b-f56a-42a0-925f-bebd4cf6abeb

![IMG-20230519-WA0070](https://github.com/elementary/triage/assets/64042095/0d72fab2-cd59-4c43-ba56-6c1876ad7f7a)
![IMG-20230519-WA0071](https://github.com/elementary/triage/assets/64042095/f6d85a9a-d54e-443a-bbb2-f9520f253c3d)
![IMG-20230519-WA0072](https://github.com/elementary/triage/assets/64042095/490abbdf-fc59-489f-9439-765f537ec60b)
![IMG-20230519-WA0073](https://github.com/elementary/triage/assets/64042095/d44c3314-1588-47dc-a03b-d1743940f946)

**Comment by leolost2605** [comment]
Hey! 
I had the same problem and for me it was because of some misconfigured default color profiles.
I think currently the easiest way to fix this is:

1. Install GNOME Control Center (a graphical utility for various settings) with the command `sudo apt install gnome-control-center`
2. Then run with `gnome-control-center` in terminal 
3. In the sidebar select "Color"
4. For each monitor there should now be an option to choose one of some preinstalled color profiles with "Default" being the default. Here you can just select one that fixes your problem and works best for you (I am not to familiar with color calibration and profiles etc. so I just selected Colorspace: sRGB which looked normal to me :)
5. You now can safely remove GNOME Control Center again to save some disk space and the selected color profile will stay applied.

I hope this helps!

**Reply by iamcalegari** [reply]
Oh, Thanks you @leolost2605 !!!
I gave up trying this distro, but I'll try this solution and come back with the results.

Thanks a lot!!!
